<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ServeripLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
